packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/calendar.tests.js

```diff
@@ -4617,6 +4618,52 @@ QUnit.module('Aria accessibility', {
         fx.off = false;
     }
 }, () => {
+    QUnit.test('aria-label should be \'Readonly-calendar\' and role = group when readOnly option is true', function(assert) {
+        const $element = this.$element;
+
+        $element.dxCalendar({
+            readOnly: true
+        }).dxCalendar('instance');
+        assert.strictEqual($element.attr('aria-label'), 'Readonly-calendar', 'aria-label is set correctly');
```

Comment:
Seems like you forgot to update asserts after last label change :)

Reply:
[28bc567](https://github.com/DevExpress/DevExtreme/pull/27898/commits/28bc5675e82786dfe12c0434627ece5fe383cf65)